### PR TITLE
created tables in DB corresponding to models and relations

### PR DESCRIPTION
**Pull request by Filari02**
database: MariaDB

to connect everyone must set these environment parameters (certain values are pinned in discord, do not share them :) )
- spring.datasource.url
- spring.datasource.username
- spring.datasource.password

### CODE REVIEW

File: Backend/src/main/java/lt/vu/ads/models/Account.java [hunk 1]
```diff
@@ -3,19 +3,30 @@
...
+import javax.persistence.*;
+
 @Getter
 @Setter
+@MappedSuperclass
 public class Account {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
```
Comment (Ugniusst) [inline Backend/src/main/java/lt/vu/ads/models/Account.java]: I am not sure, but if ID is auto generated, I think it won't be null.

File: Backend/src/main/java/lt/vu/ads/models/Address.java [hunk 2]
```diff
@@ -4,25 +4,33 @@
...
+    @Column(nullable = false)
     private Long id;
 
+    @Column(nullable = false)
     private String city;
 
+    @Column(nullable = false)
     private String street;
 
+    @Column(nullable = false)
```
Comment (Ugniusst) [inline Backend/src/main/java/lt/vu/ads/models/Address.java:26]: Maybe Flat number is needed (which is optional)?

File: Backend/src/main/java/lt/vu/ads/models/Account.java (hunk 1 above)
Comment (Filari02) [reply Backend/src/main/java/lt/vu/ads/models/Account.java]: I used Intellij auto generated code, but it should not impact anything if i delete that @Column row. Will change

File: Backend/src/main/java/lt/vu/ads/models/Address.java (hunk 2 above)
Comment (Filari02) [reply Backend/src/main/java/lt/vu/ads/models/Address.java:26]: you can write flat number in house number. for example, house number - flat number

File: Backend/src/main/java/lt/vu/ads/models/Order.java [hunk 3]
```diff
@@ -3,47 +3,67 @@
...
+    @Column(nullable = false)
     private Long id;
 
+    @Column(nullable = false)
     private int orderCode;
 
-    private OrderStatus orderStatus;
-
+    @Column(columnDefinition="BOOLEAN DEFAULT false")
     private boolean isExpress;
```
Comment (Foteno) [inline Backend/src/main/java/lt/vu/ads/models/Order.java]: instead of columnDefinition can just use `private boolean isExpress = false;`

**Review by Ugniusst — APPROVED** [review]

**Review by Foteno — APPROVED** [review]
all good

**Review by Tomas-Per — APPROVED** [review]

**Review by aule2000 — APPROVED** [review]
Good

**Pull request merged**